server/src/test/java/org/opensearch/plugins/PluginInfoTests.java

```diff
@@ -367,6 +370,31 @@ public void testSerialize() throws Exception {
         assertThat(info2.toString(), equalTo(info.toString()));
     }
 
+    public void testToXContent() throws Exception {
+        PluginInfo info = new PluginInfo(
+            "fake",
+            "foo",
+            "dummy",
+            Version.CURRENT,
```

Comment:
do we have a test around when it will return list of versions

Reply:
Currently its not feasible to create plugin info with multiple ranges. Test to be added when multiple ranges are supported.